code/game/gamemodes/cult/runes.dm

```diff
@@ -316,14 +316,22 @@ var/list/teleport_runes = list()
 	cultist_desc = "converts a normal crewmember on top of it to the cult. Does not work on loyalty-implanted crew."
 	invocation = "Mah'weyh pleggh at e'ntrath!"
 	icon_state = "3"
-	req_cultists = 2
```

Comment:
I thought the whole point of making multiple conversions is that it's a team-based game that requires cultists to work together--especially given the toxicity of stun-paper convert spam (which was very much alive and well with old-cult).